sdk/servicebus/azservicebus/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/Azure/azure-sdk-for-go/sdk/servicebus/azservicebus
```

Comment:
Are all of these services required? I don't think autorest is needed in client libraries.

Reply:
I'll have to do a cleanup here. This code doesn't even build yet (that's for the next couple of PRs). This is just a verbatim import.